kustomization/components/kube-vip/README.md

```diff
@@ -3,9 +3,10 @@
 [![Current Version](https://img.shields.io/badge/dynamic/json?style=for-the-badge&label=version&query=%24.kustomization%2Fcomponents%2Fkube-vip&url=https%3A%2F%2Fraw.githubusercontent.com%2Fmarinatedconcrete%2Fconfig%2Frefs%2Fheads%2Fmain%2F.release-please-manifest.json)](https://github.com/marinatedconcrete/config/releases?q=%22kustomize-kube-vip%22)
 [![Pod Security Standard: Privileged](https://img.shields.io/badge/pod_security_standard-privileged-red?style=for-the-badge&logo=kubernetes&logoColor=%23326CE5)](https://kubernetes.io/docs/concepts/security/pod-security-standards/)
 
-This will deploy [kub-vip](https://kube-vip.io/) as two `DaemonSet`s:
-1) for the control plane
-2) for all services' ingress IP address
+This will deploy [kube-vip](https://kube-vip.io/) as two `DaemonSet`s:
+
+1. for the control plane
+2. for all services' ingress IP address
```

Comment:
There's no typos here; it just choose to change the formatting for no good reason

Reply:
It also changed the formatting (which, IDC that much about), but it changed 'kub-vip' to 'kube-vip'